packages/overlay/src/components/Debugger.tsx

```diff
@@ -48,7 +57,31 @@ export default function Debugger({
   isOnline: boolean;
   setTriggerButtonCount: (count: NotificationCount) => void;
   fullPage: boolean;
+  setReloadSpotlight: React.Dispatch<React.SetStateAction<number>>;
 }) {
+  const isSentryIntegrationAdded = integrations.some(integration => integration.name === 'sentry');
+  const { setEvents } = useContext(SentryEventsContext);
+  const navigate = useNavigate();
+
+  const clearEvents = () => {
+    const sidecarUrl = `http://localhost:${PORT}/clear`;
```

Comment:
I think someone in theory could use a different URL than `localhost` right?
can we pass down the URL?

Reply:
yes, we can, but it would require creating a `DELETE /stream` API endpoint instead of using the `/clear` endpoint.

 At present, the user can establish a connection between the overlay and the sidecar by providing a custom sidecar stream endpoint.

Reply:
if that is ok, i'll push the changes after confirmation.

Thanks